src/main/java/uk/gov/hmcts/ccd/v2/external/resource/SupplementaryDataResource.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.hmcts.ccd.v2.external.resource;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import org.springframework.hateoas.RepresentationModel;
+import uk.gov.hmcts.ccd.domain.model.std.SupplementaryData;
+import uk.gov.hmcts.ccd.domain.model.std.SupplementaryDataRequest;
+import uk.gov.hmcts.ccd.v2.external.controller.CaseController;
+
+import static org.springframework.hateoas.server.mvc.WebMvcLinkBuilder.linkTo;
+import static org.springframework.hateoas.server.mvc.WebMvcLinkBuilder.methodOn;
+
+@Data
+@EqualsAndHashCode(callSuper = true)
+@NoArgsConstructor
+public class SupplementaryDataResource extends RepresentationModel<RepresentationModel<?>> {
```

Comment:
we no longer need to do HATEOAS so in theory we could remove this

Reply:
Done